resources/add-team-member.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```

Comment:
This is actually a script I used to keep track of my changes, but I think i'd be good to keep it to make it easier to add data to team.json.